readme.md

```diff
@@ -56,17 +76,31 @@ def hello(request):
 ```
 
 Finally, before starting up the server, create a key pair to be used by the server for signing and verifying JWT:
+
 ```commandline
 python manage.py make_rsa
 ```
+
 You should see two files created in the root of project repository:
-- jwt-signing.pem  # this is the private key used to sign a JWT, keep this secret, store appropriately
-- jwt-signing.pub  # this is the public key used to verify a JWT
 
+- jwt-signing.pem # this is the private key used to sign a JWT, keep this secret, store appropriately
+- jwt-signing.pub # this is the public key used to verify a JWT
 
 ## Documentation
 
+### Stateless authentication
+
+The `ninja_simple_jwt.auth.backends.TokenUserBackend` provides a mechanism to authenticate stateless using encoded user data from JWT to provide a custom `TokenUser` for requests to use. To make use of it, you simply need to add the backend to Django settings:
+
+```
+AUTHENTICATION_BACKENDS = [
+    "ninja_simple_jwt.auth.backends.TokenUserBackend",
+    "django.contrib.auth.backends.ModelBackend",  # Keep for username/password auth
+]
```

Comment:
Is there a reason to add this Ninja authentication in the `AUTHENTICATION_BACKENDS`? 

Ninja specifically has a different authn mechanism. If you put it here AND you put it in your router, then you are going to run this behavior twice, creating overhead in the request flow. And if you don't add it to the Ninja Router/API, then it won't be noted in the generated documentation. I would have thought that just adding it within the context of Ninja would be sufficient.